lxc/lxc-openvdc/lxc-openvdc.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/mholt/archiver"
+	"github.com/pkg/errors"
+)
+
+var lxcPath string
+var cacheFolderPath string
+var containerPath string
+var containerName string
+var rootfsPath string
+var imgPath string
+
+func main() {
+
+	_dist := flag.String("dist", "centos", "Name of the distribution")
+	_release := flag.String("release", "7", "Release name/version")
+	_arch := flag.String("arch", "amd64", "Container architecture")
+	_rootfs := flag.String("rootfs", "", "Rootfs path")
+	_containerName := flag.String("name", "", "Container name")
+	_containerPath := flag.String("path", "", "Container path")
+	_errorLogPath := flag.String("error-log-path", "/etc/openvdc/", "Error log path")
+	_imgPath := flag.String("img-path", "127.0.0.1/images", "Image path")
+	_cachePath := flag.String("cache-path", "/var/cache/lxc", "Cache path")
+
+	flag.Parse()
+
+	dist := *_dist
+	release := *_release
+	arch := *_arch
+	rootfsPath = *_rootfs
+	containerPath = *_containerPath
+	containerName = *_containerName
+	errorLogPath := *_errorLogPath
+	imgPath = *_imgPath
+	cachePath := *_cachePath
+
+	lxcPath = "/usr/share/lxc/"
```

Comment:
Can we make this more portable?

1. Detect the path from go-lxc.
2. Set the path at build time for ``lxc-openvdc``